mars/services/task/execution/ray/fetcher.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+from collections import namedtuple
+from typing import Dict, List
+from ..api import Fetcher, register_fetcher_cls
+
+
+_FetchInfo = namedtuple("FetchInfo", ["key", "object_ref", "conditions"])
+
+
+@register_fetcher_cls
+class RayObjectFetcher(Fetcher):
```

Comment:
Why not `RayFetcher` which is just as `MarsFetcher`?

Reply:
RayObject is a concept for ray. But, I am glad to rename it to RayFetcher.